src/pages/form/[id].tsx

```diff
@@ -1,21 +1,34 @@
-import { Container, Typography } from "@mui/material";
+import { api } from "../../utils/api";
+import JsonForm from "../../components/forms/JsonForm";
+import { useState, useEffect, useMemo } from "react";
 import { useRouter } from "next/router";
-import { useMemo } from "react";
 import { type JSONObject } from "superjson/dist/types";
-import ErrorPage from "../../components/error/error";
-import JsonForm from "../../components/forms/JsonForm";
 import DefaultLoadingPage from "../../components/loading/loading";
-import { api } from "../../utils/api";
+import ErrorPage from "../../components/error/error";
+import { Card, Container, Typography } from "@mui/material";
+import { useSession } from "next-auth/react";
 
 /**
  * Form page that gets the collection by id and renders the form based
  * on the schemas and uiSchemas from the collection
  */
 const Form = () => {
+  const [submitted, setSubmitted] = useState(false);
+  const [countdown, setCountdown] = useState(5);
   const router = useRouter();
+  const user = useSession();
   const { data, error, isLoading } = api.collection.getCollectionById.useQuery({
     id: router.query.id as string,
   });
+  const {
+    data: submissionData,
+    error: submissionError,
+    isLoading: submissionLoading,
+  } = api.collection.getSubmissionByCollectionId.useQuery({
+    collectionId: router.query.id as string,
+    userId: user?.data?.user?.id ?? "",
```

Comment:
This should be accessed in the trpc handler, it doesn't need to be passed in. The ctx object in the trpc handler has access to this, take a look at some other routes.